uaclient/cli.py

```diff
@@ -399,7 +399,7 @@ def main(sys_argv=None):
     args = parser.parse_args(args=cli_arguments)
     cfg = config.UAConfig()
     try:
-        log_level = int(cfg.log_level)
+        log_level = cfg.log_level
```

Comment:
If we're no longer calling `int()`, nothing here should be raising a `TypeError`; can we drop the try/except here too? (And, presumably, just refer to `cfg.log_level` directly where we're currently using `log_level` in this function?)

Reply:
I can catch that, if I catch something like this I was going to suggest using the default value (info). does that work?

Reply:
So you're suggesting that these lines in `setup_logging`: https://github.com/CanonicalLtd/ubuntu-advantage-client/blob/e74b4c1ad81b1c77a709459d2581a85e1a620c64/uaclient/cli.py#L370-L371 will change to fallback to INFO if we get a `ValueError` from `setLevel`?

That sounds reasonable to me.